priv/catalog/49591F.yaml

```diff
@@ -0,0 +1,64 @@
+id: "49591F"
+name: "sbd SBD_STARTMODE"
+group: "SBD"
+description: |
+  SBD_STARTMODE is set to the expected value
+remediation: |
+  ## Abstract
+  If not set to always, SBD will not automatically start if the node was previously fenced as it will expect the cluster in a clean state.
+  **IMPORTANT**: Always verify these steps in a testing environment before doing so in production ones!
+
+  The SBD is not used in GCP or AWS environments.
+  ## Remediation
+  Run the following commands in order:
+
+  1. Put cluster into maintenance mode:
+     ```crm configure property maintenance-mode=true```
+  2. Stop the cluster:
+     ```crm cluster stop```
+  2. Set the SBD_STARTMODE parameter to `always` on `/etc/sysconfig/sbd`:
+     ```
+     [...]
+     SBD_STARTMODE="always"
+     [...]
+     ```
+  3. Restart the cluster:
+     ```crm cluster start```
+  4. Put cluster out of maintenance mode:
+     ```crm configure property maintenance-mode=false```
+
+  ## References
+  Azure:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker#set-up-the-iscsi-target-server-sbd-device
+
+  AWS:
+
+  GCP:
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+
+facts:
+  - name: sbd_SBD_STARTMODE
+    gatherer: sbd_config
+    argument: SBD_STARTMODE
+
+values:
+  - name: expected_SBD_STARTMODE
+    # should only run on 'azure', 'nutanix' or 'kvm', NOT on 'gcp' or 'aws'
+    default: "clean"
```

Comment:
Let's remove quotes from here and next values. I see some checks with them and other without.
This is clear string, so it shouldn't be needed